documentation/docs/steps/cloudFoundryCreateService.md

```diff
@@ -2,6 +2,17 @@
 
 ## ${docGenDescription}
 
+## Prerequisites
+
+* This step is for creating one or multiple Services in Cloud Foundry.
+* Cloud Foundry API endpoint, Organization, Space and user are available
+* If you chose to create a single Service the Service Instance Name, Plan Broker of the Service to be created have to be available
```

Comment:
"Plan Broker of the Service"
I think something is missing here.
Furthermore the create-service-push plugin uses some unusual terminology regarding "Broker". To avoid confusion, I would note that you have to enter the Service Name (like in cf create-service) and refer to the documentation (like below). Or am I mistaken here?

Reply:
Seems like I was mistaken 😁. From the cf cli [documentation](http://cli.cloudfoundry.org/en-US/cf/create-service.html):
```
-b
Create a service instance from a particular broker. Required when service name is ambiguous
```
But maybe this could still be worked into the documentation

Reply:
Added it to the documentation that this flag that this flag can be set when the service name is ambiguous